utils/drop.ts

```diff
@@ -6,10 +6,12 @@ export const DEFAULT_DROP = {
 
 export const HOLDER_OF_DROP_MAP = {
   '50': '51',
+  '352': '377',
```

Comment:
TIL, we have this. is there a way to make it dynamic?

Reply:
oohh, this is for gallery item. lgtm

Reply:
> oohh, this is for gallery item.
Yes